client/dashboard/sites/overview/scan-card.tsx

```diff
@@ -16,30 +17,9 @@ const CARD_PROPS = {
 };
 
 function getScanURL( site: Site ) {
-	const domain = isSelfHostedJetpackConnected( site ) ? 'cloud.jetpack.com' : 'wordpress.com';
-	return `https://${ domain }/scan/${ site.slug }`;
-}
-
-function ScanCardUpsell( { site }: { site: Site } ) {
-	return (
-		<UpsellCard
-			heading={ __( 'Security scans' ) }
-			description={ __( 'We guard your site. You run your business.' ) }
-			externalLink={ getScanURL( site ) }
-			trackId={ CARD_PROPS.trackId }
-		/>
-	);
-}
-
-function ScanCardUnavailable() {
```

Comment:
Does this mean that it's not possible to have an unavailable state? What about when the user has Jetpack installed but Scan deactivated? Maybe they want to use a separate plugin. I'm not sure if scan.state === 'unavailable' covers that case though so maybe it's not a detectable state.

Reply:
> What about when the user has Jetpack installed but Scan deactivated?

But is that possible? I tried doing that via `/wp-admin/admin.php?page=jetpack_modules`, but Scan is not listed there even though the Jetpack plan supports that.

Reply:
I tried deactivating Protect but it seems I can still run Scan... maybe I'm misunderstanding the feature

<img width="777" height="233" alt="image" src="https://github.com/user-attachments/assets/22d5ce09-d6b5-483f-89eb-59b46d3024ac" />